source/ColorWindow.cpp

```diff
@@ -41,14 +42,15 @@
 * It is created with the dimensions of BRect
 */
 ColorWindow :: ColorWindow (BRect frame, BHandler *handler, rgb_color color)
-			: BWindow (frame, B_TRANSLATE("Change the background color"), B_TITLED_WINDOW, B_NOT_RESIZABLE) {
+			: BWindow (frame, B_TRANSLATE("Change the default background color"), B_TITLED_WINDOW, B_NOT_RESIZABLE) {
 
 	// Variables
-	BPoint 				 leftTop(10.0, 50.0);
+	BPoint 				 leftTop(10.0, 10.0);
 	color_control_layout matrix;
 	long 				 cellSide;
 
-	BButton				 *undoButton;
+	BButton				 *revertButton;
+	BButton				 *setAsDefaultButton;
 
 	// Create the  view and set the background color, then add child to window
```

Comment:
The Color window should also use BLayoutBuilder and all widgets the layout-aware versions without the BRect parameter.